acceptance-tests/src/test/java/gov/di_ipv_drivingpermit/pages/WelshLangDrivingLicencePageObject.java

```diff
@@ -0,0 +1,1015 @@
+            LOGGER.info(
+                    "Byddwn yn gwirio eich manylion gyda'r DVLA i sicrhau nad yw eich trwydded yrru wedi cael ei chanslo na'i hadrodd fel un sydd ar goll neu wedi ei dwyn.");
+        }
+    }
+
+    public void lastNameErrorSentenceWelsh() {
+        String expectedText =
+                "Gwall:\n" + "Rhowch eich enw olaf fel y mae'n ymddangos ar eich trwydded yrru";
+        String actualText = lastNameErrorMsg.getText();
+        Assert.assertEquals(expectedText, actualText);
+    }
+
+    public void firstNameErrorSentenceWelsh() {
+        String expectedText =
+                "Gwall:\n" + "Rhowch eich enw cyntaf fel y mae'n ymddangos ar eich trwydded yrru";
+        String actualText = firstNameErrorMsg.getText();
+        Assert.assertEquals(expectedText, actualText);
+    }
+
+    public void middleNameErrorSentence() {
+        String expectedText =
+                "Gwall:\n"
+                        + "Rhowch unrhyw enwau canol fel y maent yn ymddangos ar eich trwydded yrru";
+        String actualText = middleNameErrorMsg.getText();
+        Assert.assertEquals(expectedText, actualText);
+    }
+
+    public void enterDOBErrorTextWelsh() {
+        String expectedText =
+                "Gwall:\n" + "Rhowch y dyddiad fel y mae'n ymddangos ar eich trwydded yrru";
+        String actualText = issueDateErrorMsg.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void enterDOBErrorTextWelshDVA() {
+        String expectedText =
+                "Gwall:\n" + "Rhowch y dyddiad fel y mae'n ymddangos ar eich trwydded yrru";
+        String actualText = enterDateErrorDVAMsg.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void enterValidDOBErrorTextWelsh() {
+        String expectedText = "Gwall:\n" + "Gwiriwch eich bod wedi rhoi eich dyddiad geni yn gywir";
+        String actualText = enterDateErrorMsg.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void errorMessageFutureDOBWelsh() {
+        String expectedText = "Gwall:\n" + "Rhaid i'ch dyddiad geni fod yn y gorffennol";
+        String actualText = enterDateErrorMsg.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void errorMessageFutureDOBWelshDVA() {
+        String expectedText = "Gwall:\n" + "Rhaid i'ch dyddiad geni fod yn y gorffennol";
+        String actualText = enterDateErrorDVAMsg.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void enterInValidIssueDate() {
+        new DrivingLicencePageObject().LicenceIssueDay.click();
```

Comment:
Could this be updated from
new DrivingLicencePageObject().LicenceIssueDay.click();
to
DrivingLicencePageObject drivingLicencePageObject = new DrivingLicencePageObject();
drivingLicencePageObject.LicenceIssueDay.click();

So were not recreating the object every time

Reply:
Changed to DrivingLicencePageObject drivingLicencePageObject = new DrivingLicencePageObject();